slices/TableSlice/index.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div v-if="!isDefaultVariant" class="not-prose block w-full overflow-x-auto mb-8" style="width: calc(100% + 4rem); margin-left: -2rem;">
```

Comment:
not sure about those width calculations. What do you try to achieve with that? I'd just keep it full so it's in the same width as all the rest of the content

Reply:
@sopoour since the team is fine with the current design, I added the styling to maintain same width. It is not important though; we can remove the styling.

Reply:
@sopoour styling removed in latest commits